docs/terminal/melbourne.md

```diff
@@ -284,6 +296,13 @@ The following Standard Assignable Headings may be used for aircraft assigned the
 
 **When Runway 09 is in use for arrivals, all Runway 16 departures shall be assigned heading 160.
 
+### ML TCU Internal
+All aircraft transiting between internal ML TCU boundaries must be heads-up coordinated.
+
+!!! phraseology
+    <span class="hotline">**MDS** -> **MAV**</span>: "via JAYBI, JST603"  
+    <span class="hotline">**MAV** -> **MAV**</span>: "JST603, A030"
+
```

Comment:
Does this apply for all boundaries, or just those with MAV?

Reply:
> Does this apply for all boundaries, or just those with MAV?

all boundaries (we forgot to have this in there the whole time lol)